packages/plugin-packages/mashroom-portal-sandbox-app/src/js/components/MessageBusSendForm.tsx

```diff
@@ -19,13 +20,16 @@ type FormData = {
 
 type Props = {
     messageBus: MashroomPortalMessageBus;
+    portalStateService: MashroomPortalStateService;
     activePortalApp: ActivePortalApp | undefined | null;
     topicsSubscribedByApp: Array<string>;
     addMessagePublishedBySandbox: (messageBus: MessageBusMessage) => void;
 }
 
 export default class MessageBusSendForm extends PureComponent<Props> {
 
+    sbAutoTest = getQueryParams(this.props.portalStateService).autoTest;
```

Comment:
instead of reading the query params again here it would be better to read in in SandboxApp and just pass it down to the components that need it